modules/clients/src/test/java/org/apache/ignite/internal/jdbc2/JdbcStreamingSelfTest.java

```diff
@@ -160,6 +163,59 @@ protected Connection createStreamedConnection(boolean allowOverwrite, long flush
         super.afterTest();
     }
 
+    /**
```

Comment:
Do we need a test for INSERT INTO SELECT FROM scenario ? I would prefer to add.

Reply:
`java.sql.SQLException: Streaming mode supports only INSERT commands without subqueries.`